emcc.py

```diff
@@ -4167,6 +4167,8 @@ def process_libraries(state, linker_inputs):
       new_flags.append((i, flag))
       continue
     lib = strip_prefix(flag, '-l')
+    if lib == "c":
+      continue
```

Comment:
I wonder if we should instead avoid adding `native_lib` to `forced_stdlibs` on line 4180, and if we should do it for all native system libs.. do we ever want to allow the side module to include a system library?   Shouldn't they always get linking into the main module?

Perhaps we should at least do this for all default libraries (e.g. libc++, libcompiler_rt, etc)